pyo3-macros-backend/src/attributes.rs

```diff
@@ -233,13 +233,41 @@ pub fn take_attributes(
 
 pub fn take_pyo3_options<T: Parse>(attrs: &mut Vec<syn::Attribute>) -> Result<Vec<T>> {
     let mut out = Vec::new();
-    take_attributes(attrs, |attr| {
-        if let Some(options) = get_pyo3_options(attr)? {
-            out.extend(options);
+    let mut all_error = ErrorCombiner(None);
```

Comment:
Personally I prefer a plural name:

```suggestion
    let mut all_errors = ErrorCombiner(None);
```